src/engraving/libmscore/excerpt.cpp

```diff
@@ -207,7 +223,6 @@ void Excerpt::createExcerpt(Excerpt* excerpt)
     // Set instruments and create linked staves
     for (const Part* part : parts) {
         Part* p = new Part(score);
-        p->setId(part->id());
```

Comment:
Can you please explain the reason for removing this line?

Reply:
`Excerpt::parts()` returns a list of pointers to `Part`s in the master score and for each part an new `Part` is created in the `Excerpt` score. This is **not** the same `Part` as the one in the master score, hence it should have its own, unique, id.